serverless.yaml

```
@@ -15,6 +15,11 @@ custom:
   oauthRedirect: ${opt:oauthRedirect, 'http://localhost'}
   config: ${file(serverless_config.json)}
   useHapiValidator: ${opt:useHapiValidator, 'false'}
+  logLevel:
+    !If
+    - isDev
+    - 'debug'
+    - 'info'

```

```suggestion
  useHapiValidator: ${opt:useHapiValidator, 'false'}
  logLevel: ${opt:logLevel, 'error'}
```

I would suggest making it deployer's choice but default to the cheapest option ('error'). We will also need to handle the case when they input an invalid level